### ISSUE TITLE
[Bugfix] VoiceOver doesn't read inline error messages for TextField

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va-forms-system-core#448]
## Description
On recent testing, it was discovered that VoiceOver doesn't alert users of error messages when tabbing through `TextField` components and leaving them empty. This behaviour wasn't observed in other fields, like `RadioGroup` or `CheckboxField`. 

## Steps to Recreate
1. Enable VoiceOver on the `ClaimantInformation` page
1. Tab through
1. Notice how even though the error message for first name and last name shows up, it isn't read out by VoiceOver
1. Tab through the `RadioGroup` field, notice how the error message `Please provide a response` is read by VoiceOver straight away. 

## Considerations
- Unsure if this is an issue with how we've implemented `TextField` with the underlying design system web component, or an issue with the web component itself. 

## Tasks
- [ ] Fix `TextField` or file issue with DST if it's a web component bug

## Acceptance criteria
- [ ] Tasks have been complete
- [ ] PR has been merged or issue has been submitted to DST
